qa/vagrant/src/test/resources/packaging/scripts/25_tar_plugins.bats

```diff
@@ -166,44 +147,40 @@ setup() {
     assert_file_exist "$TEMP_CONFIG_DIR/elasticsearch.yml"
 
     # Checks that plugin archive is available
-    [ -e "$SHIELD_ZIP" ]
+    [ -e "$EXAMPLE_PLUGIN_ZIP" ]
 
     # Install Shield with the CONF_DIR environment variable
-    run env "CONF_DIR=$TEMP_CONFIG_DIR" /tmp/elasticsearch/bin/plugin install "elasticsearch/shield/latest" -u "file://$SHIELD_ZIP"
+    run env "CONF_DIR=$TEMP_CONFIG_DIR" /tmp/elasticsearch/bin/plugin install jvm-example -u "file://$EXAMPLE_PLUGIN_ZIP"
     [ "$status" -eq 0 ]
 
     # Checks that Shield is correctly installed
-    assert_file_exist "/tmp/elasticsearch/bin/shield"
-    assert_file_exist "/tmp/elasticsearch/bin/shield/esusers"
-    assert_file_exist "/tmp/elasticsearch/bin/shield/syskeygen"
-    assert_file_exist "$TEMP_CONFIG_DIR/shield"
-    assert_file_exist "$TEMP_CONFIG_DIR/shield/role_mapping.yml"
-    assert_file_exist "$TEMP_CONFIG_DIR/shield/roles.yml"
-    assert_file_exist "$TEMP_CONFIG_DIR/shield/users"
-    assert_file_exist "$TEMP_CONFIG_DIR/shield/users_roles"
-    assert_file_exist "/tmp/elasticsearch/plugins/shield"
+    # TODO add bin to jvm-example
+    # assert_file_exist "/tmp/elasticsearch/bin/shield"
```

Comment:
I've merged #12897 so I'll rebase this and use it to remove the shield references now.